js/main.js

```diff
@@ -0,0 +1,28 @@
+var $pokemonImg = document.querySelector('.pokemon-img')
+var randomID = getRandomID(1, 151);
+
+addEventListener('load', getPokemon(randomID));
```

Comment:
This is not waiting for the `load` event to happen, since you are calling the `getPokemon` function on this line. you can fix this by moving your function call to get a random ID on line 2 into the function block for the request. That way, you won't need a parameter for the function.